integreat_cms/static/src/js/tinymce-plugins/custom_link_input/plugin.js

```diff
@@ -143,6 +144,16 @@ import { getCsrfToken } from "../../utils/csrf-token";
                     api.disable("submit");
                 }
 
+                if (data.completions !== "") {
+                    api.disable("text");
+                    api.disable("url");
+                    api.enable("autoupdate");
+                } else {
+                    api.enable("text");
+                    api.enable("url");
+                    api.disable("autoupdate");
+                }
+
```

Comment:
So...what's this doing? And why? :)

Reply:
This enables the checkbox to automatically update an internal link only when there is a current completion, and it disables the manual inputs for a link, too in that case. Unfortunately it does not display correctly, due to a bug in tinymce